hack/verify-code-patterns.sh

```diff
@@ -16,7 +16,8 @@
 
 set -euo pipefail
 
-out="$( find . -name '*_test.go' -not -path './vendor/*' -print0 | xargs -0 grep -En 'ioutil\.TempDir' || [[ $? == 123 ]] )"
+# Disallow usage of ioutil.TempDir in tests in favor of testutil.
+out="$(grep --include '*_test.go' --exclude 'vendor/' -EIrn 'ioutil.\TempDir' || true)"
```

Comment:
This does not yet exclude the vendor directory as desired. This does for me:
```suggestion
out="$(grep --include '*_test.go' --exclude-dir 'vendor/' -EIrn 'ioutil.\TempDir' || true)"
```